app/src/main/java/com/hci3/aris/ui/balance/BalanceFragment.java

```diff
@@ -27,9 +31,22 @@ public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup c
         return inflater.inflate(R.layout.fragment_balance, container, false);
     }
 
+
     @Override
     public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceState) {
         super.onViewCreated(view, savedInstanceState);
-    }
 
+        String[] titles = new String[]{"Balance", "Enrollment", "Scholarship"};
+        TabLayout tabLayout;
+        ViewPager2 viewPager2;
+
+        tabLayout = getView().findViewById(R.id.tabBar);
+        viewPager2 = getView().findViewById(R.id.viewPager);
```

Comment:
- Inline `TabLayout` and `ViewPager2`
- Use `view` over `getView()`
  - Naka declare na ung view
  ```java
  public void onViewCreated(@NonNull View view ...
  ```

Final output:
```java
TabLayout tabLayout = view.findViewById(R.id.tabBar);
ViewPager2 viewPager2 = view.findViewById(R.id.viewPager);
```